mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -232,7 +232,7 @@ public void abortJob(JobContext originalContext, int status) throws IOException
           .executeWith(tableExecutor)
           .onFailure((output, exc) -> LOG.warn("Failed cleanup table {} on abort job", output, exc))
           .run(output -> {
-            LOG.info("Cleaning job for table {}", jobContext.getJobID(), output);
+            LOG.info("Cleaning job for table {} {}", jobContext.getJobID(), output);
```

Comment:
Good catch 👍.

I seem to remember there being some sort of analyzer annotation or something to ensure the correct number of arguments, but for the life of me can't recall. Probably wouldn't be related to this case.

Reply:
Actually, small nit:

Could you make this `LOG.info("Cleaning table {} with job id {}", jobContext.getJobID(), output);`? I personally think having both arguments one after another will be somewhat unclear, and the emphasis should be on the table being cleaned in my opinion.

The way it is now, for a table `foo` for a MR job `job_201112212038_0004`, the output would be `Cleaning job for table foo job_201112212038_0004`. To me, that personally isn't super clear.

I think `Cleaning table foo with job id job_201112212038_0004` would be much more clear to users, and would provide a simpler / potentially more intuitive string to search for in the logs (aka looking through cabana for "Cleaning table foo").

Reply:
Sounds good, i'll do it.

Reply:
done